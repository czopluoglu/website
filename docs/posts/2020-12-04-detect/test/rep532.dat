10110110011111111011
00101101001001000111
00010011001001100111
10001000001011110111
10000011011110000001
10101000001111111111
01000000111010101110
10110001111111111111
01100101010011011111
11011100001001101110
01000100111000110011
11111101101011101011
10111100101100111010
00011000011000010010
11111111111111101110
01001111101111111111
10000001011100111011
10111111110010111101
00001110010000000010
10100000011100010110
10001001010010110100
11111111100100100111
00001100011100000100
10101100010000000000
11000001011100111110
10000011010010101010
10100000001111001110
11111100001011101111
00101000001000000010
11111010011001111011
11011101110111101111
10010110011011101101
10010101111000111111
10111111010101011111
00100001100100101111
00111111011000001110
01100001100000101100
00011111101000010111
01100111110110000101
00101011111111110110
00110000011001111111
11000101010111001101
11000011010100100100
10110111111000110011
01011001011000100100
10100000101000000011
10011001010010010101
11011010111101100011
11011101111000000101
01100000001100010011
11100011011011100111
10001101000100000011
11110000101010100110
01010001111000111110
10100011011000000000
11001110011000010111
11000000101000101010
00111000000110010111
10000101001100000010
00110111110011100011
00110001111000100011
00101101100111001111
11110111101001100110
11000001110010001110
00100010101110110111
10011101110100001011
11010101111110110111
00001001010000100000
11111111110100000111
01101111111111111111
10001011010101100011
11111110111010101010
11001101101010100010
10000111001010100010
01101101000011011011
10100111101110111100
01111110111100101110
01000100001000110010
00111101010100110101
01010111010100101010
00000101010110100111
00110001101110100111
11110101111111011111
10101000001110001111
01101011010010001111
10111110010011110111
00001000011100000111
10000101010110011010
01000111111101111111
11101011011100000001
11111000101011111110
00000000010000100010
00011111000011101110
11001000101001010010
00000100011001100011
10010000110100001010
11011111111111110111
10110000100001000000
10110111010000100101
00000000110100000011
00101000100011001001
10100011111000100011
10010001001110111110
01101101111010111010
10100101111010000011
10101111000111001101
00000101001001001010
11011111110110101011
00000000000110110010
10010111110010101111
01101101101000010011
10110111101110101111
11011011001100111010
01011001110010001001
01101000111011111011
00101101101110111111
01111011011111111111
01111001001010100110
10010001111100111110
10010101100000111110
00101010111110001111
10000010001100010011
11001110011000000011
10000011110101110011
11001001011101101111
01111101101010101100
11111101111100011010
01000010001000000011
10101110000101011011
00111011000011111111
11100000001111100110
11110101001101111110
10011001011011111111
11100000001001001010
11100101111001100011
01111111011100111110
00101010011000010011
01010010111111110111
10000001001111101111
00000010001000001101
10000000111000000101
11110000100010110111
00001011101010101110
11011100111001110110
10001100101011100110
00000100111001010110
11111011111110110010
10101101111010110110
11000000110110101111
10000111111010101111
00010001101101010110
01100010101111110111
01101001001111100111
01100011101110100101
10011001101100001110
10001010001100101101
01001101011100101011
00011111101111100111
10010101111110000111
10111111001110110111
10000000100100100011
10000011110010100000
11011101110110111010
00111111111111111111
01100001010011011011
10111011110011111111
11101101101011001111
00110000000000010011
11110001011000101010
00101011000001010110
10111001110000101110
01001001011111111110
10011001011011110101
11110010110111111111
01001001000000110111
11001000011110001111
01110111110010001011
11111101011100100011
10011101011010010010
00000000001011110110
10000111110110101010
10110101011011111111
01101100011000011111
10110011011111011011
10011111100000000000
10100001010110100010
01001100111110011010
11000001111011101111
11101000011000101111
10101101010010111111
11110010111110110011
00000001011000110010
01000010001010011111
01000001010101110110
11001001111110001111
00110000000010100001
10110101111011101111
10000000000000100000
01001000100000111110
00111101011010001111
01000001001110101111
11111011110000111011
11110011111011101101
01101110101111011111
00010100000011100111
10110011111001110011
11000101110100111010
11000010001110100111
11110101101010110010
10100011011000001010
11000000001000001010
10000000001011100101
11011001011110010011
01010001010011110011
11011111100110011011
11010101111010100010
10010000001100010001
10101111111110110111
01010100111010101111
00011001111100000110
11100111101001100011
00001010001000100011
00000001000000011110
11001100011000010001
11010110111011010111
10001000000001101111
11001011101110100000
10011110100011101011
01011111001000100110
01001011111011001111
01101010110111111111
11111110110011101110
00000011001011011111
11011111110011110010
11100011111101111010
01100001101100111111
10110111111111111110
11101011101110100100
11101111111110111011
00100001011010010111
11100000111111111110
10101001111110001100
01010011101000110011
00011111011011010011
01000011110011110011
10000000101010100111
11110111110110000111
11111100110011101001
11101101011010101011
11100000001011111010
