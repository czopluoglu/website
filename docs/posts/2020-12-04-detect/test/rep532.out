-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep532.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.26
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8551
                   IDN index value:   0.7579
                           Ratio r:   0.6395

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    3    1    4    1    1    1    1    1
    2    2    3    3    1    2    3    3    2    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    4    6    7    8    9   10   15

   -----------CLUSTER  2-------------

    2   11   12   16   19

   -----------CLUSTER  3-------------

    3   13   14   17   18   20

   -----------CLUSTER  4-------------

    5
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++--+++-++---+-+---
   +d++++++---+-------+
   ++d++++-----+------+
   -++d++++--+--+-+--++
   -+++d+++--+---------
   +++++d+++----++-----
   ++++++d-+--+------+-
   ++-+++-d+---+-++-+--
   -----+++d+--+---+--+
   +-------+d++++---+--
   +--++----+d++---+++-
   -+----+--++d++-+----
   --+----+++++d--++---
   ---+-+---+-+-d++++++
   +----+-+-----+d++++-
   ---+---+---++++d+++-
   +-------+-+-++++d+-+
   -------+-++--++++d++
   ---+--+---+--+++-+d-
   -+++----+----+--++-d
 
 No cross validation for this DETECT run
